data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -698,7 +721,10 @@ Added to repository. Passes SGML validation.
 							xml:lang="grc">d</foreign></hi>: <hi rend="italics">corr.
```

Comment:
Beta code?

All of the Greek in these old files is going to be beta code and needs to be converted to Unicode. But, there are problems with those conversion scripts so be aware of those.